src/components/common/Saturation.js

```diff
@@ -42,29 +42,29 @@ export class Saturation extends React.Component {
   }
 
   render() {
-    const styles = reactCSS({
+   const styles = reactCSS({
       'default': {
-        color: {
+        color: this.props.style && this.props.style.color ? this.props.style.color : {
           absolute: '0px 0px 0px 0px',
           background: `hsl(${ this.props.hsl.h },100%, 50%)`,
           borderRadius: this.props.radius,
         },
-        white: {
+        white: this.props.style && this.props.style.white ? this.props.style.white : {
           absolute: '0px 0px 0px 0px',
           background: 'linear-gradient(to right, #fff, rgba(255,255,255,0))',
         },
-        black: {
+        black: this.props.style && this.props.style.black ? this.props.style.black : {
           absolute: '0px 0px 0px 0px',
           background: 'linear-gradient(to top, #000, rgba(0,0,0,0))',
           boxShadow: this.props.shadow,
         },
-        pointer: {
+        pointer: this.props.style && this.props.style.pointer ? this.props.style.pointer : {
           position: 'absolute',
           top: `${ -(this.props.hsv.v * 100) + 100 }%`,
           left: `${ this.props.hsv.s * 100 }%`,
           cursor: 'default',
         },
-        circle: {
+        circle: this.props.style && this.props.style.circle ? this.props.style.circle : {
```

Comment:
Instead of overwriting all of components when modifying them, it might make sense to pass them in as a separate class and merge them in like this:
```
const { color, white, black, pointer, circle } = this.props.style || {}
const styles = reactCSS({
  'default': {
    ...
  },
  'custom: {
    color,
    white,
    black,
    pointer,
    circle,
  },
}, { 'custom': !!this.props.style )
```